test/e2e/backup_script_test.go

```diff
@@ -0,0 +1,58 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+const (
+	masterNodeLabel = "node-role.kubernetes.io/master"
+)
+
+func TestBackupScript(t *testing.T) {
+	clientSet := framework.NewClientSet("")
+	masterNodes, err := clientSet.CoreV1Interface.Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: masterNodeLabel})
+	require.NoErrorf(t, err, "error while listing master nodes")
+
+	// create debug pod on first master node
+	debugNodeName := masterNodes.Items[0].Name
+	debugNodeAsArg := fmt.Sprintf("debug node/%s", debugNodeName)
+	err = exec.Command("oc", debugNodeAsArg).Run()
+	require.NoError(t, err)
+
+	// see https://www.redhat.com/sysadmin/how-oc-debug-works
+	debugPodName := debugNodeName + "-debug"
+	debugPodName = strings.ReplaceAll(debugPodName, ".", "-")
+
+	// verify no backup exist
+	cmdAsStr := "ls -l /etc/kubernetes/cluster-backup"
+	output, err := exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).CombinedOutput()
+	require.NoError(t, err)
+	outputString := string(output)
+	require.Emptyf(t, outputString, "expected no backup, found %s", outputString)
+
+	cmdAsStr = "chroot /host"
+	err = exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).Run()
+	require.NoError(t, err)
+
+	cmdAsStr = "/usr/local/bin/cluster-backup.sh"
+	err = exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).Run()
+	require.NoError(t, err)
+
+	// verify backup created
+	cmdAsStr = "ls -l /etc/kubernetes/cluster-backup"
+	output, err = exec.Command("oc", getOcArgs(debugPodName, cmdAsStr)).CombinedOutput()
+	require.NoError(t, err)
+	outputString = string(output)
+	require.NotEmptyf(t, outputString, "expected backup, found none")
```

Comment:
:) smart. When you take the output of "find", you can also reuse the function from our backup feature:
https://github.com/openshift/cluster-etcd-operator/blob/master/test/e2e/backup_test.go#L590-L617